TodoList/TodoList/Controllers/UserController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using TodoList.Data;
+using TodoList.DTOs;
+using TodoList.Service;
+namespace TodoList.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
```

Comment:
```suggestion
    public class UsersController : ControllerBase
```
để số nhiều nhé e
xem lại Restful API về qui tắc đặt tên path